lib/net.js

```diff
@@ -1327,6 +1329,15 @@ function listen(self, address, port, addressType, backlog, fd, exclusive) {
 Server.prototype.listen = function() {
   var self = this;
 
+  if (self._listenHasBeenCalled) {
+    var er = new Error('listen method has been called more than once.');
```

Comment:
are we using `let` in newly added code or are the existing `var`'s in here keeping us consistent?